tests/test_delineateit.py

```diff
@@ -26,8 +26,8 @@
     'delineateit')
 
 # Skipping all compiled model tests temporarily for feature/plugins
-pytestmark = pytest.mark.skip(
-        reason="Temporarily ignoring compiled models for feature/plugins")
+# pytestmark = pytest.mark.skip(

```

Comment:
Is this just for testing for the PR, do we plan to uncomment these again?

Reply:
This was just for this PR. Now that #1915 is merged, I need to resolve the merge conflict.